### ISSUE TITLE
Option to ignore reference-style links

### ISSUE THREAD

**Issue by rdnlsmith** [charmbracelet/glamour#183]

**Comment by Shanduur** [comment]
I will try to implemet this 😁

**Comment by Shanduur** [comment]
Looking deeper at the code, it feels like it must be done at the parser level, not renderer level, @muesli what do you think?

**Comment by muesli** [comment]
> Looking deeper at the code, it feels like it must be done at the parser level, not renderer level, @muesli what do you think?

Yeah, I don't think we can even detect these situations in glamour itself.
